desktop/linux/install/fedora.md

```diff
@@ -1,33 +1,25 @@
 ---
 description: Instructions for installing Docker Desktop on Fedora
-keywords: requirements, apt, installation, fedora, rpm, install, uninstall, upgrade, update
+keywords: fedora, rpm, update install, uninstall, upgrade, update, linux, desktop, docker desktop, docker desktop for linux, dd4l
 title: Install Docker Desktop on Fedora
 toc_max: 4
 ---
 
-To get started with Docker Desktop on Fedora, make sure you
-[meet the prerequisites](#prerequisites), then
-[install Docker Desktop](#install-docker-desktop).
+This page contains information on how to install, launch and upgrade Docker Desktop on a Fedora distribution.
 
 ## Prerequisites
 
-Your Fedora distribution must meet the [system requirements](../install.md#system-requirements) to install and launch Docker Desktop successfully.
+To install Docker Desktop successfully, you must:
+- Meet the [system requirements](../install.md#system-requirements).
+- Have a 64-bit version of either Fedora 35 or Fedora 36.
```

Comment:
(I don't have a good solution, but let me at least leave a comment 😅)

In my experience, many users "don't read", and more than once, I encountered users not having read the supported versions of a distro. Perhaps the previous approach was "too verbose", but wondering how we can make the supported versions stand out more (for those users that only give it a "casual" look and overlook this). 🤔